inference-engine/src/low_precision_transformations/src/layer_transformation.cpp

```diff
@@ -296,6 +324,22 @@ DataPrecision LayerTransformation::getDataPrecision(
 #ifdef LPT_PRINT_DEQUANTIZATION_INFO
     printDequantizationInfo(layer);
 #endif
+    std::vector<element::Type> resultPrecisions = precisions;
+    std::vector<element::Type> FQPrecisions;
+    switch (quantizationDetails.levels) {
+        case 255:
+        case 256:
+            FQPrecisions = {element::u8, element::i8};
+            break;
+        case 65535:
+        case 65536:
+            FQPrecisions = {element::u16, element::i16};
+            break;
+        case 4294967295:
+        case static_cast<size_t>(4294967296):
+            FQPrecisions = {element::u32, element::i32};
+    }
+    resultPrecisions = NetworkHelper::precisionIntersection(precisions, FQPrecisions);
```

Comment:
not used, you can remove it

Reply:
@eshoguli still used in mat_mul.cpp and weightable_layer_transformation.cpp